cloud/scaling.rst

```diff
@@ -0,0 +1,58 @@
+Scaling
+#######
+
+Citus Cloud provides self-service scaling to deal with increased load. The web interface makes it easy to either add new worker nodes or increase existing nodes' memory capacity.
```

Comment:
"or increase existing nodes' memory capacity". Not just memory right? Also CPU count/speed? Maybe just reword to "or scale up existing nodes"